manim/mobject/geometry.py

```diff
@@ -1462,8 +1517,8 @@ def construct(self):
     """
 
     def __init__(self, *vertices, color=BLUE, **kwargs):
-        VMobject.__init__(self, color=color, **kwargs)
-        # There are actually four corners, and the first one is repeated twice to form the four vertices.
+        super().__init__(color=color, **kwargs)
+        self.vertices = vertices
```

Comment:
This attribute is unnecessary and may cause confusion with `get_vertices` and will become outdated if the shape changes
```suggestion
```